workshops/migrations/0087_auto_20160519_0739.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-19 12:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+import requests
+
+
+def populate_languages(apps, schema_editor,
+        url='https://raw.githubusercontent.com/mattcg/language-subtag-registry/master/data/json/registry.json'):
```

Comment:
The [language-tag](https://github.com/OnroerendErfgoed/language-tags/) package uses this as the underlying data.
Plus it's neatly serialized to JSON.